docs/drafts/version-ranges.md

```diff
@@ -0,0 +1,49 @@
+# Operator version ranges
+
+This document explains how to specify a version range to install or update an Operator with OLM 1.0.
+
+You define an Operator's target version in its custom resource (CR) file.
+The following list describes how OLM resolves an Operator's target version, and the resulting actions:
+
+Not specifying a version in the CR
```

Comment:
I was trying to get a definition list here. I think I messed up the formatting for that.

Reply:
So, this really doesn't look like a "list", more a a set of short paragraphs.

Reply:
Agreed. The formatting didn't turn out the way I wanted it to. I will take another approach.